source/user-manual/manager/manual-integration.rst

```diff
@@ -43,6 +45,20 @@ b. For SysV Init:
 
   # service wazuh-manager restart
 
+
+Optional filters
+^^^^^^^^^^^^^^^^
+
+These fields are used by ``Integrator`` to apply certain filters to the alerts being processed and sent to the external platforms. 
+The following considerations must be taken into account when the filters are set:
+   
+   - The behavior of the ``<group>`` field when multiple values are defined is that of an OR operation. If any of the defined groups matches the alert group, the alert is processed.
```

Comment:
Consider adding this suggestion or something similar to it:
```suggestion
   - It is possible to specify multiple group names using the ``<group>`` field a with comma-separated list. If the alert's group matches any of the groups in the list, the alert is sent. Otherwise it is ignored.
```

Reply:
Done